### DISCUSSION TITLE
css position: sticky works only in layouts

### DISCUSSION THREAD

**Discussion by mihail727** [nuxt/nuxt#17921] [Questions] [answered]
I cant understand why `position: sticky` wotks only in layouts?!
Please help.... or maybe it bug....

**Comment by Megarsh** [comment, marked as answer]
This shouldn't have anything to do with nuxt.

`position: sticky` can be stingy at times, make sure that it doesn't have any parent elements with `overflow: hidden` also if it's in a flex container make sure that it's not spanning 100% of containers height.
